mobile/native/src/ln_dlc/recover_rollover.rs

```diff
@@ -0,0 +1,72 @@
+use crate::ln_dlc::node::Node;
+use anyhow::Result;
+use ln_dlc_node::node::rust_dlc_manager::channel::signed_channel::SignedChannelState;
+use ln_dlc_node::node::rust_dlc_manager::channel::Channel;
+use ln_dlc_node::node::rust_dlc_manager::Storage;
+
+impl Node {
+    /// Checks and recovers from a potential stuck rollover.
+    pub async fn recover_rollover(&self) -> Result<()> {
+        tracing::debug!("Checking if dlc channel got stuck in rollover.");
+        let channels = self.inner.channel_manager.list_channels();
+        let channel_details = match channels.first() {
+            Some(channel_details) => channel_details,
+            None => return Ok(()),
```

Comment:
Might be worth logging that everything is Ok here, or why an we return early and don't continue